introduction.md

```diff
@@ -5,3 +5,9 @@ This is code-fox doing the GitHub course from Code School.
 I also love the HTML&CSS and Ruby paths!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?